src/QsCompiler/CommandLineTool/Options.cs

```diff
@@ -17,6 +17,81 @@
 
 namespace Microsoft.Quantum.QsCompiler.CommandLineCompiler
 {
+    /// <summary>
+    /// Default values for command line options if nothing is specified. 
+    /// </summary>
+    internal static class DefaultOptions
+    {
+        public const string Verbosity = "normal";
+        public const Options.LogFormat OutputFormat = Options.LogFormat.Default;
+        public const int TrimLevel = 1;
+    }
+
+
+    public class CompilationOptions : Options
+    {
+        [Option("trim", Required = false, Default = DefaultOptions.TrimLevel, SetName = CODE_MODE,
+        HelpText = "[Experimental feature] Integer indicating how much to simplify the syntax tree by eliminating selective abstractions.")]
+        public int TrimLevel { get; set; }
+
+        [Option("load", Required = false, SetName = CODE_MODE,
+        HelpText = "[Experimental feature] Path to the .NET Core dll(s) defining additional transformations to include in the compilation process.")]
+        public IEnumerable<string> Plugins { get; set; }
+
+        [Option("target-package", Required = false, SetName = CODE_MODE,
+        HelpText = "Path to the NuGet package containing target specific information and implementations.")]
+        public string TargetPackage { get; set; }
+
+        [Option('p', "assembly-properties", Required = false, SetName = CODE_MODE,
+        HelpText = "Additional properties to populate the AssemblyConstants dictionary with. Each item is expected to be of the form \"key=value\".")]
+        public IEnumerable<string> AdditionalAssemblyProperties { get; set; }
+
+        /// <summary>
+        /// Returns a dictionary with the specified assembly properties as out parameter. 
+        /// Returns a boolean indicating whether all specified properties were successfully added.
+        /// </summary>
+        internal bool ParseAssemblyProperties(out Dictionary<string, string> parsed)
+        {
+            var success = true;
+            parsed = new Dictionary<string, string>();
+            foreach (var keyValue in this.AdditionalAssemblyProperties ?? new string[0])
+            {
+                var pieces = keyValue?.Split("=");
+                var valid = pieces != null && pieces.Length == 2;
+                success = valid && parsed.TryAdd(pieces[0].Trim().Trim('"'), pieces[1].Trim().Trim('"')) && success;
+            }
+            return success;
+        }
+
+        /// <summary>
+        /// Returns null if TargetPackage is not null or empty, and 
```

Comment:
I think this should read "Returns null if TargetPackage is null or empty" instead of "not null or empty"

Reply:
Yes, thank you!